.readme-partials.yaml

```diff
@@ -0,0 +1,104 @@
+custom_content: |
+  #### Creating a dataset
+  With BigQuery you can create datasets. A dataset is a grouping mechanism that holds zero or more
+  tables. Add the following import at the top of your file:
+
+  ```java
+  import com.google.cloud.bigquery.DatasetInfo;
+  ```
+  Then, to create the dataset, use the following code:
+
+  ```java
+  // Create a dataset
+  String datasetId = "my_dataset_id";
+  bigquery.create(DatasetInfo.newBuilder(datasetId).build());
+  ```
+
+  #### Creating a table
+  With BigQuery you can create different types of tables: normal tables with an associated schema,
+  external tables backed by data stored on [Google Cloud Storage][cloud-storage] and view tables that
```

Comment:
You'll need to make this link a non-reference link as we don't have it in the template.

Reply:
Done

Reply:
I don't see a change here. Any change you manually make in the README.md itself will be wiped on the next regeneration.

Reply:
PTAL again